cernopendata/config.py

```diff
@@ -312,6 +312,38 @@
             availability=dict(terms=dict(
                 field='distribution.availability.keyword',
                 order=dict(_term='asc'))),
+            event_number= {
+                'range': {
+                    'field': 'distribution.number_events',
+                    'ranges': [
+                        {
+                            'key' : '0 - 999',
+                            'from': 0,
+                            'to': 999
+                        },
+                        {
+                            'key' : '1000 - 9999',
+                            'from': 1000,
+                            'to': 9999
+                        },
+                        {
+                            'key' : '10 000 - 99 999',
+                            'from': 10000,
+                            'to': 99999
+                        },
+                        {
+                            'key' : '100 000 - 999 999',
+                            'from': 100000,
+                            'to': 999999
+                        },
+                        {
+                            'key' : '1 000 000 -',
```

Comment:
I think we might be able to reuse the existing `range_filter`. Can you try to use double dashes for the ranges (`--`) as the default [`range_filter`](https://github.com/inveniosoftware/invenio-records-rest/blob/b8db18e3c0a0943d99050c224ed858888d8a7595/invenio_records_rest/facets.py#L38) [expects the two dashes](https://github.com/inveniosoftware/invenio-records-rest/blob/b8db18e3c0a0943d99050c224ed858888d8a7595/invenio_records_rest/facets.py#L52). Also, are you sure the numbers should have spaces in between?